### PR TITLE
Update all.json

### PR DESCRIPTION
**Pull request by thwjanssen89**
|Site    |Urlscan| Screenshot|
|-------|-------|-------|
| fastcoiner.com | https://urlscan.io/result/110f8240-4698-45e8-8c3d-5c26d459b9e0/ | https://user-images.githubusercontent.com/97024090/195517381-40123e8f-d9e8-4d20-8b70-95ebb3d54b6c.png |
| fixedfloat.to | https://urlscan.io/result/75daafba-8c45-4153-962c-16c2621eec8c/dom/ | https://user-images.githubusercontent.com/97024090/195522896-30b4dd40-35ee-444c-91fc-2a59bf569b34.png |
| indexrect.online | NA | https://user-images.githubusercontent.com/97024090/195517986-2d74c0b4-e7e5-4e35-9f10-2af93d3aedd7.png |

### CODE REVIEW

File: all.json [hunk 1]
```diff
@@ -28,6 +28,10 @@
 	],
 	"deny": [
 		"07e96d43-381a-46a3-9c16-6daf97213efc.co",
+		"fastcoiner.com",
+		"fixedfloat.to",
```
Comment (jacogr) [inline all.json:32]: Not convinced on this one.

Comment (thwjanssen89) [reply all.json:32]: which one? Fastcoiner is a generic seedstealer and fixedfloat.to is the fake version of fixedfloat.com

Comment (jacogr) [reply all.json:32]: The issue with the fixedfloat item is that from the screenshots and the urlscan there is no definitive proof that it is less than honest. So any additional info here would be great.

Comment (thwjanssen89) [reply all.json:32]: There are over 10 submits like this with the exact same screenshot

Comment (thwjanssen89) [reply all.json:32]: This is a very well known scam. The real one is fixedfloat.com

**Review by frankywild — APPROVED** [review]

**Pull request merged**